pkg/cloudprovider/aws/instance.go

```diff
@@ -70,15 +70,15 @@ func (p *InstanceProvider) Create(ctx context.Context,
 	if err != nil {
```

Comment:
Just to double check if this `err` is non-nil, we are _guaranteed_ no instances launched?

Reply:
There should be no instances launched if an error is thrown from the request.